airflow-core.yaml

```diff
@@ -0,0 +1,149 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
+      - findutils
+      - glibc-locale-en
+      - glibc-locales
+      - gnupg-utils
+      - gzip
+      - mariadb-dev
+      - mysql
+      - openssl
+      - postgresql-client
+      - postgresql-dev
+      - python-3.12
```

Comment:
Let's create a var called `py-version`, set to 3.12, and reuse that. Will make it easier to transition to new versions of Python Also, is there a reason we've gone ahead with 3.12 here?

Reply:
Thanks @EyeCantCU , yes there's a reason as airflow most latest supported python version as we speak is 3.12 https://airflow.apache.org/blog/airflow-2.9.0
I have made the corresponding changes wrt using var `py-version`